### PR TITLE
add heal command

### PR DESCRIPTION
**Pull request by fengyuecanzhu**

### CODE REVIEW

File: src/main/java/emu/grasscutter/command/commands/FullHpCommand.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import emu.grasscutter.game.GenshinPlayer;
+import emu.grasscutter.game.props.FightProperty;
+import emu.grasscutter.server.packet.send.PacketAvatarFightPropUpdateNotify;
+import emu.grasscutter.server.packet.send.PacketAvatarLifeStateChangeNotify;
+import emu.grasscutter.server.packet.send.PacketEntityFightPropUpdateNotify;
+import emu.grasscutter.server.packet.send.PacketLifeStateChangeNotify;
+
+import java.util.List;
+
+@Command(label = "fullhp", usage = "fullhp|fhp",
```
Comment (MlgmXyysd) [inline src/main/java/emu/grasscutter/command/commands/FullHpCommand.java]: `heal` better?

Comment (fengyuecanzhu) [reply src/main/java/emu/grasscutter/command/commands/FullHpCommand.java]: > `heal` better?

yep,i'll change.

**Review by MlgmXyysd — APPROVED** [review]

File: src/main/java/emu/grasscutter/command/commands/HealCommand.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import emu.grasscutter.server.packet.send.PacketEntityFightPropUpdateNotify;
+import emu.grasscutter.server.packet.send.PacketLifeStateChangeNotify;
+
+import java.util.List;
+
+@Command(label = "heal", usage = "heal|h",
+        description = "Heal all characters in your current team.", aliases = {"h"}, permission = "player.heal")
+public class HealCommand implements CommandHandler {
+    @Override
+    public void execute(GenshinPlayer sender, List<String> args) {
```
Comment (MlgmXyysd) [inline src/main/java/emu/grasscutter/command/commands/HealCommand.java:18]: `execute` is no longer valid, use `onCommand` instead

**Review by MlgmXyysd — CHANGES_REQUESTED** [review]

File: src/main/java/emu/grasscutter/command/commands/HealCommand.java (hunk 2 above)
Comment (fengyuecanzhu) [reply src/main/java/emu/grasscutter/command/commands/HealCommand.java:18]: > `onCommand`

Where is the `onCommand`?

Comment (MlgmXyysd) [reply src/main/java/emu/grasscutter/command/commands/HealCommand.java:18]: WTF, when did they change back🤔

Comment (KingRainbow44) [reply src/main/java/emu/grasscutter/command/commands/HealCommand.java:18]: on `development` i changed it back to `execute` :p
execute is the proper term to use here

**Review by KingRainbow44 — APPROVED** [review]

**Review by MlgmXyysd — APPROVED** [review]

**Pull request merged**